src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/dto/LocationDto.kt

```diff
@@ -34,6 +39,42 @@ data class Location(
   @Schema(description = "Child Locations", required = false)
   val childLocations: List<Location>? = null,
 
+  @Schema(description = "Alternative description to display for location", example = "Wing A", required = false)
+  val description: String? = null,
+
+  @Schema(description = "Additional comments that can be made about this location", example = "Not to be used", required = false)
+  val comments: String? = null,
+
+  @Schema(description = "Sequence of locations within the current parent location", example = "1", required = false)
+  val orderWithinParentLocation: Int? = null,
+
+  @Schema(description = "Indicates the location is available", example = "true", required = true)
```

Comment:
_Enabled_ rather than _available_, if we want to use a word other that active in the description. IMO a cell could be active but be unavailable for other reasons?

Reply:
Ok Thanks

Reply:
@danbenton-mojdt updated